test/cypress/integration/pages/manage-your-college-money.js

```diff
@@ -0,0 +1,55 @@
+import { PayingForCollege } from '../../components/paying-for-college/';
+
+const page = new PayingForCollege();
+
+describe( 'Paying for College', () => {
+  beforeEach( () => {
+    page.openMoney();
+  } );
+  describe( 'Manage your college money', () => {
+    it( 'should display answers to each question', () => {
+      cy.get( '#_o1' ).click();
+      cy.get( '#_o1' ).should( 'contain', 'When should I get a bank account?' );
+      cy.get( '#answer1' ).should( 'be.visible' )
+        .and( 'contain', 'Choose an account as soon as possible' );
+
+      cy.get( '#_o2' ).click();
+      cy.get( '#_o2' ).should( 'be.visible' )
+        .and( 'contain', 'How do I avoid paying unexpected fees?' );
+      cy.get( '#answer2' ).should( 'contain', 'very few accounts charge no fees at all' );
```

Comment:
Looks like you inserted the be-visible test on the wrong item here, and as above we don't need to test what's in `#_o2`.

```suggestion
      cy.get( '#answer2' ).should( 'be.visible' )
        .and( 'contain', 'very few accounts charge no fees at all' );
```

Reply:
Done